src/lib/massa-react/hooks/useWriteSmartContract.tsx

```diff
@@ -118,7 +123,7 @@ export function useWriteSmartContract(
     let loadingToastId: string | undefined;
 
     try {
-      const publicClient = await clientFromNetwork();
+      const { publicClient, provider } = await getClientAndProvider(account);
```

Comment:
Argument of type 'IAccount' is not assignable to parameter of type 'Account'.
  Type 'IAccount' is missing the following properties from type 'Account': privateKey, publicKey, version, sealer, and 2 more.

Reply:
We have to wait for new wallet pro version